doc/translation_process.md

```diff
@@ -46,7 +46,7 @@ Visit the [Transifex Signup](https://www.transifex.com/signup/) page to create a
 You can find the Bitcoin translation project at [https://www.transifex.com/projects/p/bitcoin/](https://www.transifex.com/projects/p/bitcoin/).
 
 ### Installing the Transifex client command-line tool
-The client it used to fetch updated translations. If you are having problems, or need more details, see [http://docs.transifex.com/developer/client/setup](http://docs.transifex.com/developer/client/setup)
+The client it used to fetch updated translations. If you are having problems, or need more details, see [https://docs.transifex.com/client/client-configuration](https://docs.transifex.com/client/client-configuration)
```

Comment:
Why this url? Seems the current url equivalent is: https://docs.transifex.com/client/installing-the-client
Based on: https://web.archive.org/web/20161011000624/http://docs.transifex.com/client/setup/

Reply:
Fixed, thx